CMakeLists.txt

```diff
@@ -40,6 +40,7 @@ link_directories(${CMAKE_BINARY_DIR}/lib)
 
 add_subdirectory(deps)
 add_subdirectory(lib)
+add_subdirectory(lib-ziti-tunnel-cbs)
```

Comment:
could we break this out into subdirectories under lib/?

```
lib/
  ziti-tunnel/
  ziti-tunnel-cbs/
  ...
```
  I'll be adding another library for the ip-service code very soon.